spec/helpers/map_locations_helper_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+describe MapLocationsHelper do
+  context "#related_proposals_coordinates" do
+    it "returns related proposals by any tag for given proposal" do
+      create(:tag, :category, name: "culture")
+      create(:tag, :category, name: "sports")
```

Comment:
I think these two lines are not necessary; the `tag_list=` method automatically creates tags if they don't exist.